python/cudf/cudf/tests/test_orc.py

```diff
@@ -783,6 +786,36 @@ def test_orc_writer_decimal(tmpdir, scale):
     assert_eq(expected.to_pandas()["dec_val"], got["dec_val"])
 
 
+def test_orc_reader_multiple_files(datadir):
+
+    path = datadir / "TestOrcFile.testSnappy.orc"
+
+    df_1 = pd.read_orc(path)
+    df_2 = pd.read_orc(path)
+    df = pd.concat([df_1, df_2], ignore_index=True)
+
+    gdf = cudf.read_orc([path, path], engine="cudf").to_pandas()
+
+    assert_eq(df, gdf)
+
+
+def test_orc_reader_multi_file_single_stripe(datadir):
+
+    path = datadir / "TestOrcFile.testSnappy.orc"
+
+    # should raise an exception
+    with pytest.raises(ValueError):
+        cudf.read_orc([path, path], engine="cudf", stripes=[0])
+
+
+def test_orc_reader_multi_file_multi_stripe(datadir):
+
+    path = datadir / "TestOrcFile.testStripeLevelStats.orc"
+    gdf = cudf.read_orc([path, path], engine="cudf", stripes=[[0, 1], [2]])
+    assert gdf.shape[0] == 11000
+    assert gdf.shape[1] == 2
```

Comment:
Why not `assert_eq()` ?

Reply:
just wasn't paying attention. Made the change.